### ISSUE TITLE
forwardedRef incorrectly typed

### ISSUE THREAD

**Issue by oscar-b** [FortAwesome/react-fontawesome#393]
**Describe the bug**
The prop `forwardedRef`, introduced here: https://github.com/FortAwesome/react-fontawesome/pull/341/files#diff-7aa4473ede4abd9ec099e87fec67fd57afafaf39e05d493ab4533acc38547eb8R24 seems to be incorrectly typed compared to React:

```
Type '((instance: unknown) => void) | MutableRefObject<unknown> | null' is not assignable to type '((e: any) => void) | RefObject<any> | undefined'.
  Type 'null' is not assignable to type '((e: any) => void) | RefObject<any> | undefined'.
```

The `ref` prop from `React.forwardRef` is typed like this:
```
(parameter) ref: ((instance: unknown) => void) | React.MutableRefObject<unknown> | null
```
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L565

The `forwardedRef` prop from `<FontAwesomeIcon />` expects the prop to be optional, not possibly null.

**Reproducible test case**
https://codesandbox.io/s/confident-http-bpf4y?file=/src/Icon.tsx

**Expected behavior**
No type error.

**Desktop (please complete the following information):**
```
"@fortawesome/react-fontawesome": "^0.1.11",
"react": "^16.13.1",
```

**Additional context**

**Comment by TiagoPortfolio** [comment]
Any update on this?

**Comment by robmadole** [comment]
PR's are welcome on this. We don't use TypeScript so we lean pretty heavily on the community to update this.

**Comment by TiagoPortfolio** [comment]
> PR's are welcome on this. We don't use TypeScript so we lean pretty heavily on the community to update this.

I opened a PR #396

**Comment by robmadole** [comment]
Thanks @TiagoPortfolio . Merged and released 0.1.13.
